dev/dll/Microsoft.UI.Xaml.vcxproj

```diff
@@ -28,16 +27,6 @@
     <DisableEmbeddedXbf Condition="'$(Configuration)'=='Release'">false</DisableEmbeddedXbf>
     <AppxBundlePlatforms>x86|x64|arm|arm64</AppxBundlePlatforms>
   </PropertyGroup>
-  <!-- Temporary hack: This should really go into the product-common props/targets file -->
-  <PropertyGroup Condition="$(BuildingWithBuildExe) == 'true'">
-    <DefaultBuildArchName>x86</DefaultBuildArchName>
-    <BuildArchName Condition="'$(Platform)' == 'Win32'">x86</BuildArchName>
-    <BuildArchName Condition="'$(Platform)' == 'x64'">amd64</BuildArchName>
-    <BuildArchName Condition="'$(Platform)' == 'arm'">arm</BuildArchName>
-    <BuildArchName Condition="'$(Platform)' == 'arm64'">arm64</BuildArchName>
-    <BuildArchName Condition="'$(Platform)' == 'AnyCPU'">$(DefaultBuildArchName)</BuildArchName>
-    <BuildArchName Condition="'$(BuildArchName)' == ''">$(DefaultBuildArchName)</BuildArchName>
-  </PropertyGroup>
   <PropertyGroup Condition="'$(Use64BitLinker)' == 'true' AND '$(BuildingWithBuildExe)' == 'true'">
```

Comment:
As long as you're removing these kinds of blocks, I assume checks like this should go too? And then you should remove all the $(BuildingWithBuildExe) != 'true" as well. ... But really cleanup should be separated from functional changes. Can you do the cleanup separately?